Dockerfile.task_base

```diff
@@ -9,31 +9,6 @@ ENV PIP_NO_CACHE_DIR=1
 
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
-RUN apt-get update && apt-get install -y software-properties-common
-
-RUN add-apt-repository ppa:ubuntugis/ppa && \
-     apt-get update && \
-     apt-get install -y build-essential python3-dev python3-pip \
-     jq unzip ca-certificates wget curl git && \
-     apt-get autoremove && apt-get autoclean && apt-get clean
-
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
-
-# See https://github.com/mapbox/rasterio/issues/1289
-ENV CURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
-
-# Install Python 3.8
-RUN curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh" \
-   && bash "Mambaforge-$(uname)-$(uname -m).sh" -b -p /opt/conda \
-   && rm -rf "Mambaforge-$(uname)-$(uname -m).sh"
-
-ENV PATH /opt/conda/bin:$PATH
-ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
-
-RUN mamba install -y -c conda-forge python=3.8 gdal pip setuptools cython numpy
```

Comment:
AFAICT, we aren't using gdal directly. Only via rasterio. I'm 100% sure what was happening before, but I think we were building rasterio from source, which IIRC requires at least some GDAL files to be present.

Now we're getting rasterio from its wheel, which statically links to its own copy of GDAL.

Reply:
I think this has been carried forward for a long while based on some rasterio installation issues with original `etl`, which seems to have been solved with newer versions of rasterio.